src/components/ErrorBoundary/ErrorHandlingWrapper.tsx

```diff
@@ -0,0 +1,31 @@
+'use client';
+
+import { QueryErrorResetBoundary } from '@tanstack/react-query';
+import { ComponentType, ReactNode, Suspense } from 'react';
+
+import ErrorBoundary, { FallbackProps } from './ErrorBoundary';
+
+interface PropsType {
```

Comment:
P4; ErrorHandlingWrapperPropsType 으로 제안드려봅니다~
```suggestion
interface ErrorHandlingWrapperPropsType {
```

Reply:
감사합니다 수정했습니다!